### PR TITLE
Update allowselfservicepurchase-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Added syntax to update service point manager security protocol to TLS1.2 (tested)

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2681

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
You can also upgrade to TLS 1.2. For more details, see [How to enable TLS 1.2](https://docs.microsoft.com/mem/configmgr/core/plan-design/security/enable-tls-1-2).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks.

**Comment by cmcatee-MSFT** [comment]
@yogkumgit I'm waiting on confirmation from the PM before I approve this PR.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT  Have we got any update, Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder.

**Pull request merged**
